src/SUMMARY.md

```diff
@@ -43,6 +43,8 @@
   - [Server](net/server.md)
 - [Operating System](os.md)
   - [External Command](os/external.md)
+- [Mathematics](mathematics.md)
```

Comment:
So I'd like this to the be **Science** category to match the Crates.io category "Science"  I'd also rather group all the **Mathematics** into a sub-section of the science category rather than breaking them into calculus/geometry/etc.

Reply:
To make sure I understand what you mean, change the titles from:

```
16. Mathematics
    16.1. Calculus
```

to:

```
16. Science
    16.1. Mathematics
```
?

Reply:
yes